.github/workflows/tests-rs-package.yml

```diff
@@ -49,10 +51,15 @@ jobs:
         if: inputs.package == 'random-document'
         uses: ./.github/actions/javy
 
+      - name: Setup Trunk for GroveDBG build
+        uses: ./.github/actions/trunk
+
       - uses: clechasseur/rs-clippy-check@v3
         with:
           args: --package ${{ inputs.package }} --all-features --locked -- --no-deps
         env:
+          TRUNK_TOOLS_WASM_OPT: version_117
+          TRUNK_TOOLS_WASM_BINDGEN: 0.2.92
```

Comment:
Not sure we want to hardcode versions in CI

Reply:
we have hardcoded bindgen in several places as I remember

Reply:
Very possible. Versions are matching?

Reply:
yes, doesn't compile otherwise